planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -634,6 +636,11 @@ void planning_scene_monitor::PlanningSceneMonitor::excludeRobotLinksFromOctree()
       if (h)
         link_shape_handles_[links[i]].push_back(std::make_pair(h, j));
     }
+    if(!warned && ((ros::Time::now() - start) > ros::Duration(30.0)))
+    {
+      ROS_WARN_STREAM_NAMED("PlanningSceneMonitor::excludeRobotLinksFromOctree()", "It is likely there are too many vertices in collision geometry");
```

Comment:
Sorry, I just noticed now.
Please change the first parameter to `planning_scene_monitor`.
The name can be used to selectively enable/disable loggers and we already use the short name in the class.